lib/candy_check/play_store/product_acknowledgements/acknowledgement.rb

```diff
@@ -0,0 +1,45 @@
+module CandyCheck
+  module PlayStore
+    module ProductAcknowledgements
+      # Verifies a purchase token against the PlayStore API
+
+      class Acknowledgement
+        # @return [String] the package_name which will be queried
+        attr_reader :package_name
+        # @return [String] the item id which will be queried
+        attr_reader :product_id
+        # @return [String] the token for authentication
+        attr_reader :token
+
+        # Initializes a new call to the API
+        # @param package_name [String]
+        # @param product_id [String]
+        # @param token [String]
+        def initialize(package_name:, product_id:, token:, authorization:)
+          @package_name = package_name
+          @product_id = product_id
+          @token = token
+          @authorization = authorization
+        end
+
+        def call!
+          acknowlege!
+
+          CandyCheck::PlayStore::ProductAcknowledgements::Response.new(
+            result: @response[:result], error_data: @response[:error_data])
+        end
+
+        private
+
+        def acknowlege!
+          service = CandyCheck::PlayStore::AndroidPublisherService.new
```

Comment:
I think this class shouldn't have the remote call. Looking at the architecture of the gem, this simply encapsulates the response and works as an entity.

Reply:
Perhaps it os the other class that can be skipped.